templates/config.yaml

```diff
@@ -122,6 +177,65 @@ templates:
           <<: *kyma_generic_component
           path: components/kyma-operator
           since: "1.7"
+          optional: true
+      - to: ../prow/jobs/kyma/tests/console-backend-service/console-backend-service-tests-generic.yaml
+        values:
+          <<: *kyma_generic_component
+          path: tests/console-backend-service
+          since: "1.6"
+      - to: ../prow/jobs/kyma/tests/application-connector-tests/application-connector-tests-generic.yaml
+        values:
+          <<: *kyma_generic_component
+          path: tests/application-connector-tests
+          since: "1.7"
+          optional: true
+      - to: ../prow/jobs/kyma/tests/application-gateway-tests/application-gateway-tests-generic.yaml
+        values:
+          <<: *kyma_generic_component
+          path: tests/application-gateway-tests
+          since: "1.7"
+          optional: true
+      - to: ../prow/jobs/kyma/tests/application-operator-tests/application-operator-tests-generic.yaml
+        values:
+          <<: *kyma_generic_component
+          path: tests/application-operator-tests
+          since: "1.7"
+          optional: true
+      - to: ../prow/jobs/kyma/tests/application-registry-tests/application-registry-tests-generic.yaml
+        values:
+          <<: *kyma_generic_component
+          path: tests/application-registry-tests
+          since: "1.7"
+          optional: true
+      - to: ../prow/jobs/kyma/tests/compass-runtime-agent/compass-runtime-agent-tests-generic.yaml
+        values:
+          <<: *kyma_generic_component
+          path: tests/compass-runtime-agent
+          since: "1.7"
+          optional: true
+      - to: ../prow/jobs/kyma/tests/connection-token-handler-tests/connection-token-handler-tests-generic.yaml
+        values:
+          <<: *kyma_generic_component
+          path: tests/connection-token-handler-tests
+          since: "1.7"
+          optional: true
+      - to: ../prow/jobs/kyma/tests/connector-service-tests/connector-service-tests-generic.yaml
+        values:
+          <<: *kyma_generic_component
+          path: tests/connector-service-tests
+          since: "1.7"
+          optional: true
+      - to: ../prow/jobs/kyma/tests/console-backend-service/console-backend-service-tests-generic.yaml
+        values:
+          <<: *kyma_generic_component
+          path: tests/console-backend-service
+          since: "1.6"
+      - to: ../prow/jobs/kyma/tests/end-to-end/upgrade/upgrade-tests-generic.yaml
```

Comment:
I think this one is duplicated

Reply:
Indeed. I removed the other and adjusted this one.